core/settings.py

```diff
@@ -300,3 +301,4 @@
 MAX_IMAGE_DIMENSIONS = (5000, 5000)
 ALLOWED_IMAGE_MODES = ["RGB", "RGBA", "L"]
 
+TEST_RUNNER = 'django.test.runner.DiscoverRunner'
```

Comment:
Explicitly setting the test runner is fine, but unless you have custom requirements, this is Django's default. Is this line necessary?